docker/project.go

```diff
@@ -15,7 +15,7 @@ import (
 const ComposeVersion = "1.5.0"
 
 // NewProject creates a Project with the specified context.
-func NewProject(context *Context) (project.APIProject, error) {
+func NewProject(context *Context, parseOptions *config.ParseOptions) (*project.Project, error) {
```

Comment:
Same as in #260, changing the return `project.APIProject` to `*project.Project`. I really think it's a better idea to return an abstraction of that struct since we can then provide a Mock on the tests and be able to asses correct functionality. New methods can be added to the interface if they need to be accessed from outside the package